pages/04.Reference/13.identity-and-access/docs.en.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Identity and Access
+published: true
+date: '20-03-2020 10:40'
+taxonomy:
+    category:
+        - docs
+---
+
+## Overview
+
+OpenStack's Identity API is implemented via Keystone. It provides API client authentication, service discovery, and distributed multi-tenant authorization.
+
+## Password
+
+The common way for a user or service to authenticate towards Keystone is by using username and password. You may enter the user credentials in the OpenStack dashboard, or use them inside of your software to manage your OpenStack ressources. A sample OpenStack RC file which can be used for the OpenStack python CLI client would be the following.
+
+```shell
```

Comment:
I believe we have a How-To or tutorial for that? if yes maybe we link it from here.

In the reference guide I would instead expect a short description of the Authentication flow (e.g. Retrieve token from Keystone, send request to Nova with the token, Nova checks the token using keystone, Nova sends response) and a table with supported methods. In that table we should also mention that we don't support two factor authentication yet.

Reply:
So we describe : https://docs.openstack.org/keystone/queens/api_curl_examples.html